From c6197f05ef6d19d072847ae628c4de65799743e1 Mon Sep 17 00:00:00 2001
From: Chisoon Jeong <chisoon.jeong@lge.com>
Date: Thu, 26 Mar 2020 21:53:05 +0000
Subject: [PATCH] Add '#include <string>' in trace_event_memory_overhead.h for
 clarity.

Depending on compilation environment, trace_event_memory_overhead.h|cc
is not compilable, owing to missing <string> header.

Change-Id: I8e52e7bae33e68d5dc3cc08b1d38fcd3c87303a6
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2117932
Reviewed-by: dsinclair <dsinclair@chromium.org>
Reviewed-by: Scott Violet <sky@chromium.org>
Reviewed-by: Kentaro Hara <haraken@chromium.org>
Commit-Queue: Nico Weber <thakis@chromium.org>
Cr-Commit-Position: refs/heads/master@{#753781}
---
 base/trace_event/trace_event_memory_overhead.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base/trace_event/trace_event_memory_overhead.h b/base/trace_event/trace_event_memory_overhead.h
index 69468d464098..e10a3eca667d 100644
--- a/base/trace_event/trace_event_memory_overhead.h
+++ b/base/trace_event/trace_event_memory_overhead.h
@@ -8,6 +8,7 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#include <string>
 #include <unordered_map>
 
 #include "base/base_export.h"
-- 
2.25.2

